cadcdata/cadcdata/storageinv.py

```diff
@@ -538,11 +618,11 @@ def cadcremove(self, id):
                 duration = time.time() - start
                 logger.debug('{} removed in {} ms'.format(id, duration))
                 return
-            except (exceptions.HttpException, socket.timeout) as e:
+            except exceptions.TransferException as e:
                 # try a different URL
```

Comment:
Should there be an `if isinstance....` addition to the `urls` here as well?

Also, should the `except` clause catch `Exception`?

Reply:
There are no streaming involved so there's no `TransferException`. Changed it to a `Exception` instead.